charts/datafold/charts/portal-client/values.yaml

```diff
@@ -0,0 +1,43 @@
+replicaCount: 1
+image:
+  repository: gcr.io/datadiff-mm/datafold-server-onprem
```

Comment:
The gcr would be really old and deprecated on google. Are we sure we don't have a us.docker.dev type of url for this?  GCR is practically gone.

Reply:
Oh, I see now - will test it out.

I noticed we have the same thing on df-shell, but interestingly, I had the new code available in the portal-client pod that I deployed via this image.

however I don't see the [image here](https://console.cloud.google.com/artifacts/docker/datadiff-mm/us/datafold?hl=en&inv=1&invt=AbiHDw&project=datadiff-mm), so I don't know where it comes from
<img width="996" alt="image" src="https://github.com/user-attachments/assets/c90ce10c-63e4-48cf-90db-d5049377518c">

Reply:
![image](https://github.com/user-attachments/assets/b5327046-e094-411c-aee5-953606ae1e0b)
![image](https://github.com/user-attachments/assets/1b0ffea7-ebe1-49de-a4ed-ed51f20d8798)

These were already used, as this image.repository value wasn't used when computing the image path:
>           image: "{{ .Values.global.shellRepository }}/server:{{ .Values.global.datafoldVersion }}"